tools/check_format.py

```diff
@@ -635,7 +729,13 @@ def checkFormatVisitor(arg, dir_name, names):
   # python lists are passed as references, this is used to collect the list of
   # async results (futures) from running checkFormat and passing them back to
   # the caller.
-  pool, result_list = arg
+  pool, result_list, owned_directories, error_messags = arg
+
+  # Sanity check CODEOWNERS.  This doesn't need to be done in a multi-threaded
+  # manner as it is a small and limited list.
+  if dir_name.startswith('./source/extensions/') and '/' in dir_name[20:]:
```

Comment:
Any way to make the 20 and 9 magic constants less magical either via some more descriptive code or some comments? It's not immediately clear to me what is going on here.